src/main/java/io/camunda/testing/assertions/BpmnAssertions.java

```diff
@@ -0,0 +1,26 @@
+package io.camunda.testing.assertions;
+
+import io.camunda.zeebe.client.api.response.ProcessInstanceEvent;
+import org.camunda.community.eze.RecordStreamSource;
+
+public abstract class BpmnAssertions {
+
+  static ThreadLocal<RecordStreamSource> recordStreamSource = new ThreadLocal<>();
+
+  public static void init(RecordStreamSource recordStreamSource) {
+    BpmnAssertions.recordStreamSource.set(recordStreamSource);
```

Comment:
:x: I think we should also clear this after a test. Otherwise we accumulate objects on the heap, and/or the next test on the same thread could run against the record stream of the test before it. This would be very hard to detect